### ISSUE TITLE
Minishift start fails when providing ISO URL with http://

### ISSUE THREAD

**Issue by robnester-rh** [minishift/minishift#1650]
**Issue:**
Providing a direct URL to an ISO image when executing `minishift start` fails

**Expected results:**
Minishift starts with the specified ISO file

**Actual results:**

```
$ minishift start --iso-url http://artifacts.ci.centos.org/fedora-atomic/minishift/iso/minishift.iso

-- Checking if requested hypervisor 'kvm' is supported on this platform ... OK
-- Checking if KVM driver is installed ... 
   Driver is available at /usr/local/bin/docker-machine-driver-kvm ... 
   Checking driver binary is executable ... OK
-- Checking if Libvirt is installed ... OK
-- Checking if Libvirt default network is present and active ... OK
-- Checking the ISO URL ... FAIL
   See the 'Managing Minishift' topic for more information
```

**Additional info**
```
$ minishift version
minishift v1.8.0+6455fc3
```

**Comment by gbraad** [comment]
This is probably the issue: 

  https://github.com/minishift/minishift/blob/master/cmd/minishift/cmd/start_preflight.go#L455

it drops in the else case too soon, and assumes this has to be file://, which means the prefix http?:// is not checked

**Comment by gbraad** [comment]
As a workaround, download the image first and use `file://` instead to specify the image

**Comment by hferentschik** [comment]
Merged by pull request #1652

**Comment by gbraad** [comment]
Just noticed: but the commit description "Issue #1650 Adding test for Windows and simplifying setup" should have included mention of "iso-url". Hopefully next time...
